src/components/SearchBar/SearchBar.jsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from "react";
+import { getTopics } from "../../utils/util-api-calls";
+import { capitaliseTopic } from "../../utils/util-data-formatters";
+import "./SearchBar.css"
+
+function SearchBar({ setSearchParams }) {
+	const [topics, setTopics] = useState([]);
+
+	useEffect(() => {
+		getTopics()
+			.then((res) => {
+				setTopics(res.data.topics);
+			})
+			.catch((error) => {
+				console.log(console.error());
+			});
+	}, []);
+
+	function handleTopicSelect(event) {
+		const selectedTopic = event.target.value;
+		setSearchParams((current) => {
+			return { ...current, topic: selectedTopic };
+		});
+	}
+
+
```

Comment:
Great that this list of topics is informed by a call to the DB, so if any categories ever got added to the db, they would appear here without any changes needed to this code